app.py

```diff
@@ -69,4 +145,8 @@ def hello():
     return 'Hello, ' + s  + '!\n'
      
 if __name__ == '__main__':
+    success = initialize()
+    if not success:
+        print(f'Could not connect to services, HTTP: 503')

```

Comment:
There wouldn't be a 503 here as there is no request.  This is just starting the webserver

Reply:
yea, I was curious how I would handle . I'll modify that to just a regular error statement.